path/_os.ts

```diff
@@ -1,29 +1,8 @@
+// deno-lint-ignore-file no-explicit-any
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
 
-// Keep this up-to-date with Deno.build.os
-/**
- * Operating system type, equivalent to the type of
- * {@linkcode https://deno.land/api?s=Deno.build | Deno.build.os}.
- */
-type OSType =
-  | "darwin"
-  | "linux"
-  | "windows"
-  | "freebsd"
-  | "netbsd"
-  | "aix"
-  | "solaris"
-  | "illumos"
-  | "android";
-
-function getOsType(): OSType {
-  // deno-lint-ignore no-explicit-any
-  return (globalThis as any).Deno?.build.os ||
-    // deno-lint-ignore no-explicit-any
-    ((globalThis as any).navigator?.userAgent.includes("Win")
-      ? "windows"
-      : "linux");
-}
-
-export const isWindows: boolean = getOsType() === "windows";
+// Check Deno, then the remaining runtimes (e.g. Node, Bun and the browser)
+export const isWindows: boolean =
+  (globalThis as any).Deno?.build.os === "windows" ||
+  (navigator as any).platform?.startsWith("Win");
```

Comment:
Probably it's better to add `|| false` at the end to not make this `undefined` (In cloudflare workers both `globalThis.Deno` and `navigator.platform` are undefined)

Reply:
Done